instructor/templates/instructor/assignment_select_variables.html

```diff
@@ -1,7 +1,9 @@
 {% extends "instructor/assignment_sidebar.html" %}
 
 {% block right_side %}
-
+<script>
+var created = "{{ treatments_created }}"
```

Comment:
After you change `treatments_created` to use `json.dumps`, you should be able to make this `var created = {{ treatements_created }};` without the quotes. Also add a semicolon please :)